test-crawler/Makefile

```diff
@@ -3,4 +3,14 @@ run:
 	go run main.go config.go
 
 copy-output:
-	go run main.go config.go > ../frontend/apps/web/src/tests.json
\ No newline at end of file
+	go run main.go config.go > ../frontend/apps/web/src/tests.json
+
+run_bcg:
+	go run . -crawl=src
+
+go-unit-test:
+	go test ./...
+
+go-integration-test:
+	chmod -R 777 integrations_tests/*.sh
```

Comment:
I'm not sure this command is required - once the files in this folder are committed to git, and pulled to a local machine, their permissions should be pulled with them.
I will checkout this branch and test it on my machine.

Reply:
I pulled the repo, and ran the integration test, and everything works as expected. You can remove the line.

Yes, adding permissions through `chmod -R 777` and `chmod +x` is required when creating an executable file. However, once they are committed, you can see that you do not need to repeat this process.